docs/README.md

```diff
@@ -83,7 +83,7 @@ $ PRODUCTION=1 jekyll build
 
 ## API Docs (Scaladoc, Javadoc, Sphinx, roxygen2, MkDocs)
 
-You can build just the Spark scaladoc and javadoc by running `./build/sbt unidoc` from the `$SPARK_HOME` directory.
+You can build just the Spark scaladoc and javadoc by running `./build/sbt clean unidoc` from the `$SPARK_HOME` directory.
```

Comment:
I can imagine adding this to the docs, as users might have stale compiled code locally or something. How often would it occur in run-tests? In PR builders it starts with a clean build and I'd guess users recompile before tests. I'm also worried about slowdown for the PR builders.

Reply:
@srowen Thanks for the suggestion. I will revert the changes here.